examples/auth/openapi_swagger.json

```diff
@@ -74,55 +94,35 @@
       }
     }
   },
-  "definitions": {
-    "listShelvesResponse": {
-      "properties": {
-        "shelves": {
-          "items": {
-            "$ref": "#/definitions/shelf"
-          },
-          "type": "array"
-        }
-      },
-      "type": "object"
-    },
-    "shelf": {
-      "properties": {
-        "name": {
-          "type": "string"
-        },
-        "theme": {
-          "type": "string"
-        }
-      },
-      "required": [
-        "name",
-        "theme"
-      ]
-    }
-  },
+  "produces": [
+    "application/json"
+  ],
+  "schemes": [
+    "https"
+  ],
   "securityDefinitions": {
     "auth0_jwk": {
       "authorizationUrl": "",
       "flow": "implicit",
       "type": "oauth2",
+      "x-google-audiences": "bookstore.endpoints.cloudesf-testing.cloud.goog",
       "x-google-issuer": "123456789-compute@developer.gserviceaccount.com",
-      "x-google-jwks_uri": "https://www.googleapis.com/service_accounts/v1/jwk/123456789-compute@developer.gserviceaccount.com",
-      "x-google-audiences": "bookstore.endpoints.cloudesf-testing.cloud.goog"
+      "x-google-jwks_uri": "https://www.googleapis.com/service_accounts/v1/jwk/123456789-compute@developer.gserviceaccount.com"
     },
     "firebase": {
       "authorizationUrl": "",
       "flow": "implicit",
       "type": "oauth2",
+      "x-google-audiences": "apiproxy-231719",
```

Comment:
Also update them?

Reply:
We can, but there's no point. I just needed to update the `host` field so we can re-deploy them to `cloudesf-testing`. All the other fields can be set to anything, it doesn't matter.